api/src/api/graphql/translate/dictionary.clj

```diff
@@ -38,3 +39,70 @@
                         :label  ""
                         :roles  []
                         :frames []}))}))
+
+(defn pos->schema [pos]
```

Comment:
Lets stick to POS naming as it is used in GF. See `morphology` schema for supported values